internal/commands/fix.go

```diff
@@ -0,0 +1,52 @@
+package commands
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/eikendev/hackenv/internal/options"
+	"github.com/eikendev/hackenv/internal/scripts"
+)
+
+// FixCommand struct
+type FixCommand struct {
+	CreateBridge bool `short:"c" name:"create-bridge" help:"Create bridge"`
+	RemoveBridge bool `short:"r" name:"remove-bridge" help:"Remove bridge"`
+	Labels       bool `short:"l" name:"labels"        help:"Fix SElinux labels"`
```

Comment:
You can make each of these their own command, so they all have their own `Run()` function. This would make it a bit cleaner and the CLI directly shows the three scripts as subcommands. As inspriation, [here are the subcommands](https://github.com/pushbits/cli/blob/main/internal/application/mod.go) for the `application` command of pbcli.

Reply:
Will do this thanks